src/constants.rs

```diff
@@ -126,6 +126,7 @@ pub enum Chattype {
     Single = 100,
     Group = 120,
     VerifiedGroup = 130,
+    MailingList = 140,
```

Comment:
as these constants are used in all uis, i would hesitate to add another type here. we also have no type for "device chat" or for "saved messages". i think the "main type" visible from outside should just be "Group" therefore. we already have a "can_send" flag that is checked by the ui.

Reply:
Done.